setup.py

```diff
@@ -1,5 +1,4 @@
-#!/usr/bin/env python
```

Comment:
Since you're removing the shebang, you can also remove the executable permission from setup.py